compiler/util/clangUtil.cpp

```diff
@@ -1191,6 +1191,17 @@ void prepareCodegenLLVM()
   info->FPM_postgen = fpm;
 
   info->FPM_postgen->doInitialization();
+
+  if(ffloatOpt == 1)
+  {
+    llvm::FastMathFlags FM;
+    FM.setNoNaNs();
+    FM.setNoInfs();
+    FM.setNoSignedZeros();
+    FM.setAllowReciprocal();
+    FM.setUnsafeAlgebra();
+    info->builder->SetFastMathFlags(FM);
```

Comment:
I had to change it to setFastMathFlags for LLVM 4. Could you fix it in the PR too?